lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,54 @@
+import 'whatwg-fetch'; // fetch polyfill
```

Comment:
You probably copy/pasted this comment from somewhere, but it seems somewhat unnecessary?